config/7.0/obdemo-bank/ig/scripts/groovy/ProcessDetachedSig.groovy

```diff
@@ -2,11 +2,62 @@ import org.forgerock.http.protocol.*
 import org.forgerock.json.jose.*
 import org.forgerock.json.jose.jwk.store.JwksStore.*
 import org.forgerock.json.JsonValueFunctions.*
+import groovy.json.JsonSlurper
+
+import com.nimbusds.jose.*;
+import com.nimbusds.jose.crypto.*;
+import com.nimbusds.jose.jwk.*;
+
+def validateUnencodedPayload(jws, payload) {
+    Payload detachedPayload = new Payload(payload);
+    JWSObject parsedJWSObject = JWSObject.parse(jws, detachedPayload);
+    JWKSet jwkSet = JWKSet.load(new URL(routeArgJwkUrl));
```

Comment:
What happens if the JWKSet isn't loaded ok?

Reply:
Have added exception handling for JWKS load